guides/store-retrieve-messages.md

```diff
@@ -38,20 +38,21 @@ In order to interact with the Waku network, you first need a Waku instance:
 ```js
 import { Waku } from 'js-waku';
 
-const wakuNode = await Waku.create();
+const wakuNode = await Waku.create({ bootstrap: true });
 ```
 
-# Connect to Other Peers
-
-The Waku instance needs to connect to other peers to communicate with the network.
-You are free to choose other methods to bootstrap and DappConnect will ship with new bootstrap mechanisms in the future.
-
-For now, the easiest way is to connect to Status' Waku fleet:
+Passing the `bootstrap` option will connect your node to predefined Waku nodes hosted by Status.
+If you want to bootstrap to your own nodes, you can pass an array of multiaddresses instead:
 
 ```js
-import { getStatusFleetNodes } from 'js-waku';
-const nodes = await getStatusFleetNodes();
-await Promise.all(nodes.map((addr) => waku.dial(addr))); 
+import { Waku } from 'js-waku';
+
+const wakuNode = await Waku.create({
+  bootstrap: [
```

Comment:
Here it might make sense to have a comment saying these nodes come from Status.

As inspiration, see Geth and how they encode EF as a comment among many https://github.com/ethereum/go-ethereum/blob/master/params/bootnodes.go#L87-L91